main.tf

```
@@ -3,7 +3,60 @@ provider "aws" {
   access_key = var.access_key
   secret_key = var.secret_key
   region     = var.region
+
+}
+terraform {
```

Add a terraform version requirement. Looks like you are using .14
```suggestion
terraform {
  required_version = "> 0.14.0"
```